functions/Remove-DbaDbUser.ps1

```diff
@@ -204,11 +194,15 @@ function Remove-DbaDbUser {
 
     process {
         if ($InputObject) {
-            Remove-DbUser $InputObject
+            $server = Connect-SqlInstance -SqlInstance $InputObject.Parent.Parent.Name
```

Comment:
This method won't suffice as it does not have the full context of the connection to the instance. It will be missing connection information such as the login used to connect.

![image](https://user-images.githubusercontent.com/11204251/57998918-7c956680-7aa1-11e9-8caa-e4491bd196f5.png)

Reply:
oh wait, this is an easy fix

 $server = $InputObject.Parent.Parent